From: Peter Michael Green <plugwash@raspbian.org>
Subject: Automatically generated patch (2.2.0.33+dfsg-10+rpi1)

Last (up to) 3 git changes, FYI:

commit 5e23fdcb08db0ec628fffa42b44116ff344bb22e
Author: Peter Michael Green <plugwash@raspbian.org>
Date:   Thu Sep 28 00:06:53 2017 +0000

    Remove old raspbian patch that no longer applies, dgit will generate a new one.

commit c3dde68640b7ec709868df402d7516bd3e4d88aa
Merge: be1551c 2f61873
Author: Peter Michael Green <plugwash@raspbian.org>
Date:   Thu Sep 28 00:03:13 2017 +0000

    Manual merge of version 2.2.0.33+dfsg-9+rpi1 and 2.2.0.33+dfsg-10 to produce 2.2.0.33+dfsg-10+rpi1

commit 2f61873f0b0daf0e6e9e0668f108254d563debc9
Merge: a93e7eb bc15b1b
Author: Jochen Sprickerhof <jspricke@debian.org>
Date:   Sat Sep 9 15:17:46 2017 +0100

    Merge openni2 (2.2.0.33+dfsg-10) import into refs/heads/workingbranch
---

--- openni2-2.2.0.33+dfsg.orig/ThirdParty/PSCommon/BuildSystem/CommonDefs.mak
+++ openni2-2.2.0.33+dfsg/ThirdParty/PSCommon/BuildSystem/CommonDefs.mak
@@ -6,21 +6,21 @@ ifndef CFG
     CFG = Release
 endif
 
-# find out the platform on which we're running
-MACHINE = $(shell uname -m)
-ifneq (,$(findstring x86_64,$(MACHINE)))
-	HOST_PLATFORM = x64
-else ifneq (,$(findstring x86,$(MACHINE)))
-	HOST_PLATFORM = x86
-else ifneq (,$(findstring i686,$(MACHINE)))
-	HOST_PLATFORM = x86
-else ifneq (,$(findstring i386,$(MACHINE)))
-	HOST_PLATFORM = x86
-else ifneq (,$(findstring armv6l,$(MACHINE)))
+## find out the platform on which we're running
+#MACHINE = $(shell uname -m)
+#ifneq (,$(findstring x86_64,$(MACHINE)))
+#	HOST_PLATFORM = x64
+#else ifneq (,$(findstring x86,$(MACHINE)))
+#	HOST_PLATFORM = x86
+#else ifneq (,$(findstring i686,$(MACHINE)))
+#	HOST_PLATFORM = x86
+#else ifneq (,$(findstring i386,$(MACHINE)))
+#	HOST_PLATFORM = x86
+#else ifneq (,$(findstring armv6l,$(MACHINE)))
 	HOST_PLATFORM = Armv6l
-else
-	HOST_PLATFORM = generic
-endif
+#else
+#	HOST_PLATFORM = generic
+#endif
 
 # now check if this is a cross-compilation or not
 ifeq "$(PLATFORM)" ""
--- openni2-2.2.0.33+dfsg.orig/ThirdParty/PSCommon/BuildSystem/Platform.Arm
+++ openni2-2.2.0.33+dfsg/ThirdParty/PSCommon/BuildSystem/Platform.Arm
@@ -4,7 +4,7 @@ export GLUT_SUPPORTED=1
 ifeq "$(CFG)" "Release"
 
     # Hardware specifying flags
-    CFLAGS += -Wno-unused-local-typedefs -march=armv7-a -mtune=cortex-a9 -mfpu=neon #-mfloat-abi=softfp #-mcpu=cortex-a8
+    CFLAGS += -Wno-unused-local-typedefs #-march=armv7-a #-mtune=cortex-a9 #-mfpu=neon #-mfloat-abi=softfp #-mcpu=cortex-a8
 
     # Optimization level, minus currently buggy optimizing methods (which break bit-exact)
     CFLAGS += -O3 -fno-tree-pre -fno-strict-aliasing
